# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/mochitest/mochitest.ini
# Commit: e55eb174b342
# Full Hash: e55eb174b342c4ded8f84304980a866ba810eabd
# Author: Peter Van der Beken <peterv@propagandism.org>
# Date: 2021-12-16 16:31:55
# Regressor Bug: 1742865
# File Overlap Count: 1
# Description:
#   Bug 1742865 - Handle meta refresh correctly with session history in the parent. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132837
# ==============================================================================

diff -r cfb3ff7ce730 -r e55eb174b342 docshell/test/mochitest/mochitest.ini
--- a/docshell/test/mochitest/mochitest.ini	Thu Dec 16 11:12:53 2021 +0200
+++ b/docshell/test/mochitest/mochitest.ini	Thu Dec 16 08:19:45 2021 +0000
@@ -125,6 +125,11 @@
 support-files =
   file_bug1741132.html
 skip-if = toolkit == "android" && !sessionHistoryInParent
+[test_bug1742865.html]
+support-files =
+  file_bug1742865.sjs
+  file_bug1742865_outer.sjs
+skip-if = toolkit == "android" && debug && fission && verify # Bug 1745937
 [test_bug1743353.html]
 support-files =
   file_bug1743353.html